samples/deploy/crd/v1/ApisixRoute.yaml

```diff
@@ -803,6 +803,22 @@ spec:
                         required:
                           - serviceName
                           - servicePort
+                      plugins:
+                        type: array
+                        items:
+                          type: object
+                          properties:
+                            name:
+                              type: string
+                              minLength: 1
+                            enable:
+                              type: boolean
+                            config:
+                              type: object
+                              x-kubernetes-preserve-unknown-fields: true # we have to enable it since plugin config
```

Comment:
Incomplete comment?

Reply:
Also, it looks like we don't add this field for the `plugins` of HTTP route.

Reply:
Thanks for your review! I just copy this field from HTTP route: https://github.com/apache/apisix-ingress-controller/blob/deb044039e889e88310523f369833bb137c0e145/samples/deploy/crd/v1/ApisixRoute.yaml#L733-L745

Three are three `plugin` field defiend for HTTP route in this yaml before my change. So, may be keep the same for Stream route?

Reply:
We can keep it consistent for now

Reply:
> We can keep it consistent for now

Any suggestion to make it better? I also think copy and paste are ugly. But I don't know how to improve it.

Reply:
We can improve comments in subsequent PRs

But for CRD configuration, they are the same, direct copy-paste is ok